Security/Fundamentals/Security_Principles.mediawiki

```diff
@@ -26,77 +28,87 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
-* Allow OpenSSH, RDP connections from any host on any network.
-* Allow tools remotely executing code on every system from a centralized location (single Puppet Master, Ansible Tower, Nagios, etc. instance).
+* Establish service users with access to multiple services.
+* Allow tools remotely executing code on systems from a centralized location (single Puppet Master, Ansible Tower, Nagios, etc. instance) across multiple services.
+* Use the same TLS termination point (e.g. an HTTP reverse proxy or a load balancer) across multiple services.
+* Re-use functionality across services when not required (such as sharing load balancers, databases, etc.)
 
 == Patch Systems ==
 ''The act of updating software, in particular in order to deploy fixes to security vulnerabilities that are found in software over time.''
 
 '''Do'''
-* Establish regular patching time windows.
-* Ensure systems can be turned off and back on without affecting availability.
-* Turn on automatic patching where possible.
+* Establish regular recurring maintenance windows in which to patch software.
+* Ensure individual systems can be turned off and back on without affecting service availability.
+* Enable automatic patching where possible.
 
 == Meet Web Standards ==
-''The act of following the Mozilla Web Standard, which consist of HTTP headers, specific web-server setup, specific [https://www.w3.org/ W3C] security features, etc.''
+''The act of following the Mozilla Web Standard, which covers HTTP headers, web-server setup, and [https://www.w3.org/ W3C] security features.''
 
 '''Do'''
-* Achieve B+ or higher on [https://observatory.mozilla.org Mozilla's Observatory]
+* Achieve B+ or higher on [https://observatory.mozilla.org Mozilla's Observatory].
 * Follow the [[Security/Guidelines/Web_Security]] recommendations.
 
 == Encrypt sensitive data ==
-''The process of converting information or data into a code, especially to prevent unauthorized access.''
+''The process of converting information or data into a code to prevent unauthorized access.''
 
 '''Do'''
-* Encrypt data at rest when not actively used by the system.
-* Use full-disk encryption where available.
-* Encrypt credentials storage databases (Ansible Vault, CredStash, etc.)
-* Encrypt traffic in transit (TLS).
+* Use full-disk encryption where available on systems without physical security (laptops and mobile phones).
+* Encrypt credentials storage databases (Ansible Vault, Credstash, etc.)
+* Encrypt data in transit (during transmission).
+* Also encrypt data in transit inside the internal network.
 
-''Do not'''
-* Terminate TLS for all services in one location, then transmit the information in clear-text in the rest of the network.
+'''Do not'''
+* Terminate TLS (e.g. with a reverse proxy or load balancer) outside a system and then transmit the data in clear-text across the rest of the network.
 * Use STARTTLS without also disabling clear-text connections.
 
```

Comment:
do not - redirect users from the http to https, disable the http port (if not necessary)